src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -139,12 +141,11 @@
         path (:uri request)
         method (:request-method request)
         authentic? (true? (get-in request ring/is-authentic-key))]
-    (str "Forbidden request: " (if name
-          (format "%s(%s)" name ip)
-          ip) " access to " path " (method " method ")"
-         (if-let [ file (:file rule) ]
-           (str " at " file ":" (:line rule)))
-         (format " (authentic: %s)" authentic?))))
+    (str "Forbidden request: " (if name (format "%s(%s)" name ip) ip)
+         " access to " path " (method " method ")"
+         (if-let [file (:file rule)] (str " at " file ":" (:line rule)))
+         (format " (authentic: %s) " authentic?)
+         (format "denied by rule '%s'." (:name rule)))))
```

Comment:
I just tacked on the string `" denied by rule 'foo rule'."` to the existing message